.github/workflows/RELEASE.yaml

```diff
@@ -264,3 +270,15 @@ jobs:
             bundle/default-bundle/target/connectors-bundle-sbom.xml
             connectors-bundle-templates-${{ github.event.release.tag_name }}.tar.gz
             connectors-bundle-templates-${{ github.event.release.tag_name }}.zip
+
+  helm-deploy:
```

Comment:
@ev-codes can we rather invoke/integrate the already [existing action](https://github.com/camunda/connectors/blob/main/.github/workflows/INTEGRATION_TEST.yml#L4) instead of duplicating it? 

Docs:  https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow

Reply:
@sbuettner does the change I just pushed address your concerns?

Please don't merge yet as I need to test this new configuration.

Reply:
@sbuettner I'm going to handle the Helm integration test in another PR.

As we discussed elsewhere, the Helm integration test will be more complicated to implement to work with patch versions.